frontend/packages/client/src/pages/Community.js

```diff
@@ -140,6 +140,9 @@ export default function Community() {
 
   const history = useHistory();
 
+  const [activeWidth, setActiveWidth] = useState(105);
```

Comment:
105 is a little magic number here but its the default width of the default tab (proposals). We could make this dynamic but for sake of simplicity went with this

Reply:
is this ok for mobile devices etc? If it's filling the width of the tab I'm guessing yes ..